nyancat/README.md

```diff
@@ -6,14 +6,16 @@ Most IRISnet validators may know the Fuxi Testnet before mainnet was an early in
 
 ## How to claim Nyancat Testnet rewards
 
-Please refer to [reward-claims](./reward-claims/README.md)
+~~Please refer to [reward-claims](./reward-claims/README.md)~~
+
+Since v0.16, we will use the [HTLC](./v0.16/README.md#htlc-tasks) to claim testnet rewards.
```

Comment:
Starting from v0.16, participants will be asked to claim their testnet rewards through [atomic swaps](./v0.16/README.md#htlc-tasks)